### ISSUE TITLE
missing api-ms-win-core-winrt-l1-1-0.dll

### ISSUE THREAD

**Issue by kithub20** [lay295/TwitchDownloader#312]
### Checklist

- [X] I have checked the [issue page](https://github.com/lay295/TwitchDownloader/issues) for duplicates
- [X] I am running the latest version ([download here](https://github.com/lay295/TwitchDownloader/releases/latest))

### Describe your issue here

Hello, I downloaded, installed, and started the latest version currently available (1.40.7) on a Windows 7 SP1 system. And get each time on attempt to start the TwitchDownloader software this error popup modal window (attached screenshot to this ticket), which is about a missing dll "api-ms-win-core-winrt-l1-1-0.dll". Any idea? Thanks for looking into it.
![error](https://user-images.githubusercontent.com/75302602/163802658-0565ff22-d721-4fb3-9e71-865bcd14e903.png)

**Comment by lay295** [comment]
When you download the zip, are you extracting it fully into it's own folder? I believe this error can happen if you just extract the .exe without all of the accompanying files and dlls.

**Comment by kithub20** [comment]
I extracted it fully in its own folder. However, that missing dll is anyway not part of the zip file's content. When I google for that file, it gives the impression that it's a common issue. I dont know if it's allowed here to put links, but I found another github issue discussion, where someone seem to have solved it through adjusting something in the build configuration. 
Other people add this dll to a folder on Windows somewhere, but I dont want to download this file from insecure sources on the internet. And it would be great if that can be resolved properly. Perhaps this adjusted build configuration when creating the release zip file would be the way to go.
